### DISCUSSION TITLE
CPU time for building/Loading SFINCS models w/ v1.1.0

### DISCUSSION THREAD

**Discussion by lgrimley** [Deltares/hydromt_sfincs#221] [Q&A]
I had originally built a model using an older version of Hyromt-sfincs (e.g., back when the subgrid files was sfincs.sbg). I tried building this same model with the newest release of Hydromt-sfincs (v1.1.0). Everything mostly works but with some data renaming issues I had to solve. The one thing I noticed is that creating the SFINCS model took over 2.5 times as long-- and this wasn't just because of setting up the subgrid. I also noticed that reading the model into python takes a **very** long time and trying to write new boundary condition files doesn't really work. Curious to know if this is something others are experiencing?

**Comment by Leynse** [comment]
@roeldegoede ?

**Reply by roeldegoede** [reply]
Haven't noticed this before no, but that sounds like something we should investigate further ... I'll ask around whether other people had similar experiences.
